src/runtime/scss/tabs.scss

```diff
@@ -56,6 +56,7 @@
 }
 
 .yfm-tab-panel {
+    position: relative;
```

Comment:
Could you explain this change? Look like that will break existing styles.

Reply:
Somehow without `position: relative` it breaks page layout and extends page height by the height of content of invisible tabs.
<img width="300" alt="image" src="https://github.com/diplodoc-platform/tabs-extension/assets/15200559/46882879-3a2a-454c-86dd-fae2d9ce1ad6">
<img width="300" alt="image" src="https://github.com/diplodoc-platform/tabs-extension/assets/15200559/ad581738-61ba-4b4b-a20b-0d9cbc6a7cfb">

But when adding `position: relative` - it renders as expected:

<img width="300" alt="image" src="https://github.com/diplodoc-platform/tabs-extension/assets/15200559/be2716d3-2d4b-467c-838e-23e587060306">
<img width="500" alt="image" src="https://github.com/diplodoc-platform/tabs-extension/assets/15200559/7b2d1463-78c5-447d-a798-205b6fc827df">